WebRoot/public/login.jsp

```diff
@@ -394,7 +397,7 @@ if (application.getInitParameter("offlineMode") != null) {
 	</c:redirect>
 </c:if>
 
-<c:if test="${(empty param.client or param.client eq 'touch') and touchSupported and touchLoginPageExists}">
+<c:if test="${(empty param.client or param.client eq 'touch') and touchSupported and touchLoginPageExists and zimbraxSupported ne true}">
```

Comment:
this gives me an impression that we are ignoring client type passed from somewhere else, can we change client type only instead of doing this?

Reply:
Done

Reply:
Have done the checking condition at top